packages/sdk/client/package.json

```diff
@@ -25,6 +25,11 @@
       "browser": "./dist/lib/browser/halo/index.mjs",
       "node": "./dist/lib/node-esm/halo/index.mjs"
     },
+    "./edge": {
```

Comment:
we should think about how edge is exposed from/integrated with client overall, cc @dmaretskyi